android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -113,9 +113,9 @@ public void setEmail(@Nullable String email) {
     }
 
     @ReactMethod
-    public void updateEmail(String email) {
+    public void updateEmail(String email, @Nullable String authToken) {
```

Comment:
There should be new method for the one's with authToken right? Old method should still be there. Otherwise it will be a breaking change

Reply:
Added! Forgot I had to do that.